Sources/SLIP.swift

```diff
@@ -23,6 +23,7 @@ public struct SLIP {
         case vechain = 818
         case callisto = 820
         case wanchain = 5718350
+        case icon = 888888 //Temporary
```

Comment:
This temporary id 888888 repeats several time

Reply:
Yes it's a temporary value since ICON doesn't have a "Coin type" in the SLIP-0044 (https://github.com/satoshilabs/slips/blob/master/slip-0044.md) 
Not sure what should I use instead ?

Reply:
I added ICX in the SLIP-0044 (https://github.com/satoshilabs/slips/pull/397)